examples/user_interfaces/embedding_in_tk_canvas.py

```diff
@@ -1,63 +0,0 @@
-# -*- noplot -*-
```

Comment:
oops just realized you were talking about the whole file and not just the `noplot` line...I just meant `noplot` won't be needed with sphinx gallery